### ISSUE TITLE
Authorization on _INBOX.> 

### ISSUE THREAD

**Issue by ramasa** [nats-io/nats.net.v1#237]
Please provide option to specify custom inbox prefix (say _INBOX.client1) , so that below authorization rule would prevent client from listening to other's response (_INBOX.>)

```
authorization {
    CLIENT1 = {
        publish = ["req.foo", "req.bar"]
        subscribe = "_INBOX.client1.>"
    }
```

Server is using java client and our customer using .NET 
More context here https://github.com/nats-io/java-nats/issues/192

**Comment by sixlettervariables** [comment]
If I understand you correctly, you'd like to be able to change the inbox prefix per client; perhaps like so:
```csharp
var opts = ConnectionFactory.GetDefaultOptions();
opts.InboxPrefix = $"_INBOX.{clientId}.";
// ... other connection options
```

**Comment by ramasa** [comment]
Yes, or by default use the nats username ( _INBOX.<nats_username> )?

**Comment by ColinSullivan1** [comment]
Resolved by #322
